exercises/joseph_torsney/ex02_oo_basics/src/square.h

```diff
@@ -0,0 +1,16 @@
+#pragma once
+
+#include "shape.h"
+
+class Square: public Shape {
+
+  public:
+    Square(double side);
+
+    double getArea() const override;
+
+    double getPerimeter() const override;
+
+  private:
+    const int m_side;
```

Comment:
I think this should be a double since the variable is the length of the side. This is causing the warning on the windows build as it is trying to convert the inputted double to an int here.